crates/workspace/src/settings.rs

```diff
@@ -0,0 +1,60 @@
+mod indent_style;
+mod indent_width;
+// TODO: Can we pick a better crate name for `line_ending` so these don't collide?
+#[path = "settings/line_ending.rs"]
+mod line_ending_setting;
```

Comment:
Mildly annoying. We have a `line_ending` crate but I've also added `settings/line_ending.rs` as a mod and I need to use both in here 😢 .